apps/web/src/pages/Applications/ApplicantDashboardPage/components/DashboardHeading.tsx

```diff
@@ -186,164 +193,195 @@ const DashboardHeading = ({ user }: DashboardHeadingProps) => {
-              id: "B1Pa0D",
+                "The apprenticeship program is a part of a larger digital talent initiative in the Government of Canada that aspires to recruit awesome digital talent like yourself. This profile provides access to other digital opportunities and recruitments beyond the apprenticeship program, if you choose to apply. If you’d prefer to stick to the apprenticeship program exclusively, that’s okay too!",
+              id: "+im7hW",
               description:
                 "Second paragraph for applicant dashboard notification welcoming an IAP user",
             })}
           </p>
           <p>
-            {intl.formatMessage({
-              defaultMessage:
-                "When you log into the account you created, you’ll be taken to this page from now on. You can use this page to review your Indigenous Apprenticeship application and track your position in the program.",
-              id: "uSAdtr",
-              description:
-                "Third paragraph for applicant dashboard notification welcoming an IAP user",
-            })}
+            {intl.formatMessage(
+              {
+                defaultMessage:
+                  "When you log into your account, you’ll start on this page from now on. You can use the <strong><a>Track your applications</a></strong> section on this page to review your Indigenous apprenticeship application and track your status in the program.",
+                id: "RzNVWf",
+                description:
+                  "Third paragraph for applicant dashboard notification welcoming an IAP user",
+              },
+              {
+                a: (chunks: React.ReactNode) =>
+                  buildScrollToLink("track-applications-section", chunks),
+              },
+            )}
           </p>
         </Alert.Root>
       )}
       <div
         data-h2-display="base(grid)"
-        data-h2-grid-template-columns="base(100%) l-tablet(repeat(2, minmax(0, 1fr)))"
+        data-h2-grid-template-columns="base(100%) p-tablet(repeat(2, minmax(0, 1fr)))"
         data-h2-gap="base(x1)"
       >
         <HeroCard
           asNav
           color="secondary"
           title={intl.formatMessage({
-            defaultMessage: "My profile information",
-            id: "XvfBqb",
+            defaultMessage: "Profile information",
+            id: "gTjLic",
             description: "applicant dashboard card title for profile card",
           })}
           href={paths.profile(user.id)}
         >
-          <HeroCardProfileItem
-            sectionName={intl.formatMessage(navigationMessages.aboutMe)}
-            status={deriveSectionStatus(
-              aboutSectionHasEmptyRequiredFields(user),
-              aboutSectionHasEmptyOptionalFields(user),
-            )}
-            href={paths.aboutMe(user.id)}
+          <StatusItem
+            asListItem
+            title={intl.formatMessage({
```

Comment:
I didn't use the shared `navigationMessages` for this page.  I'm not sure these changes are supposed to be dispersed yet.